themes-default/slim/static/js/home/display-show.js

```diff
@@ -414,4 +424,19 @@ MEDUSA.home.displayShow = function() { // eslint-disable-line max-lines
             log.error(error.data);
         });
     });
+
+    function setQuality(quality, seriesSlug, episodes) {
```

Comment:
Please use `const setQuality = (quality, seriesSlug, episodes) => {`

Reply:
Are you sure? That's a really ugly syntax IMO 😕

Reply:
It feels like that at first. But it gets better when your getting used to it